addons/web/static/tests/helpers/mock_server.js

```diff
@@ -165,7 +192,8 @@ var MockServer = Class.extend({
             }
             if (!(fieldName in record)) {
                 if ('default' in model.fields[fieldName]) {
-                    record[fieldName] = model.fields[fieldName].default;
+                    const def = model.fields[fieldName].default;
+                    record[fieldName] = typeof def === 'function' ? def.call(this) : def;
```

Comment:
why would you want `default` to be a function? I don't like this

Reply:
This is the same in Python. The need is to return values that must be re-computed for every record, such as uuid, token, or "now" date fields. It also works for values that depend on env (as a general term), such as the current partner doing the operation.

Reply:
ok